### PR TITLE
Dedicated cluster vertical scaling

### PR DESCRIPTION
**Pull request by gemma-shay**
Documented workflow for increasing storage per node and increasing or decreasing compute per node.
Closes DOC-2004

### CODE REVIEW

**Comment by jseldess** [comment]
I think we should consider renaming these pages from "Cluster Management" to "Manage a CockroachDB Dedicated Cluster" and "Manage a CockroachDB Serverless Cluster".

**Review by jramos-crl — APPROVED** [review]
Looks good to me. I like that we clearly note that storage can't be reduced in a big red banner - good call out.

File: cockroachcloud/create-your-cluster.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ When scaling up your cluster, it is generally more effective to increase node si
 
 For more detailed disk performance numbers, see the relevant [GCP](https://cloud.google.com/compute/docs/disks/performance) and [AWS](https://aws.amazon.com/ebs/features/#Amazon_EBS_volume_types) documentation.
 
-To change the hardware configuration after the cluster is created, [contact Support](https://support.cockroachlabs.com).
+To change the hardware configuration after the cluster is created, see [Cluster management](cluster-management.html).
```
Comment (ianjevans) [inline cockroachcloud/create-your-cluster.md:89]:
```suggestion
To change the hardware configuration after the cluster is created, see [Manage a CockroachDB Dedicated Cluster](cluster-management.html).
```

**Review by ianjevans — APPROVED** [review]
Just a nit about the link's new title.

**Pull request merged**